modules/eco-resuming-node-maintenance-cli.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-resuming-node-maintenance-cli_{context}"]
+= Resuming a node from maintenance mode in the CLI
```

Comment:
If we first show CLI option on how put node unde maintenance, then I would move this subsection right after "Setting a node to maintenance mode in the CLI".